### DISCUSSION TITLE
No code folding in VS code

### DISCUSSION THREAD

**Discussion by mrleblanc101** [intlify/bundle-tools#46] [Ideas]
I'm not sure if this is an issue with Vetur or if this plugin need to provide some data to Vetur but I'm not able to use code folding inside `<i18n>` custom blocks. 

These blocks only contain JSON/YML so they should support code folding exactly like standalone file. Vetur support code folding for SFC inside `<template>`, `<script>` and `<style>`

**Comment by kazupon** [comment]
you can use volar, so you got the syntax highlight. 
https://github.com/johnsoncodehk/volar
